packages/libs/api/icy-graphql-client/src/client/client.ts

```diff
@@ -0,0 +1,93 @@
+import {
+  ApolloClient,
+  from,
+  HttpLink,
+  InMemoryCache,
+  NormalizedCacheObject,
+} from '@apollo/client/core';
+import { setContext } from '@apollo/client/link/context';
+import { onError } from '@apollo/client/link/error';
+import { NFTQueries } from '../queries/nft/nftQueries';
+import { IcyGraphQLClient } from './icyGraphQLClient';
+
+export interface IcyClientArguments {
+  apiKey?: string;
+}
+
+const ICY_GRAPHQL_CLIENT_SUPPRESS_WARNINGS =
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
probably just worth getting rid of this ESLINT rule if we're going to do this?

Reply:
yeah I think that's fair, will add `todo` comment to keep track

Reply:
applied on 084d13a!